CRM/Mailing/Selector/Browse.php

```diff
@@ -264,6 +258,12 @@ public function &getRows($action, $offset, $rowCount, $sort, $output = NULL) {
           'qs' => 'mid=%%mid%%&continue=true&reset=1',
           'title' => ts('Continue Mailing'),
         ],
+        CRM_Core_Action::UPDATE => [
+          'name' => ts('Re-Use'),
```

Comment:
How hard would it be to make this label adapt?

This is maybe a bit nitpicky, but "Re-Use" suggests that the mailing has been "used" before. A draft hasn't been used  before.

If it's not too hard, maybe something like this would be more apropos:

```php
  'name' => $thisItemIsADraft ? ts('Copy') : ts('Re-Use'),
```

Reply:
Agreed, I suggested [in the issue](https://lab.civicrm.org/dev/mail/-/issues/90) changing the text to "Copy". My thought is that it would be better to keep the label consistent for all mailings and just change it to "Copy" throughout. Otherwise, users may wonder what the difference is between "Copy" and "Re-use". Copy is also consistent with events, contribution pages, price sets, etc which all use some variation of Copy, Make a Copy, Copy Price Set, etc.

Reply:
Yup. "Copy" is a small UI change in near term (*and we might want to tweak the docs after this PR*), but I think you're right - "Copy" is more consistent in the big picture. It also matches the `Copy of ...` text.